theme/static/js/resourceLandingAjax.js

```diff
@@ -1044,24 +1045,44 @@ function move_to_folder_ajax_submit(source_paths, target_path) {
             }
         },
         error: function(xhr, errmsg, err){
-            display_error_message('File/Folder Moving Failed', xhr.responseText);
+            if (xhr.status == 300) {
+                $('#resp-message').text(xhr.responseText);
+                $("#source_paths").val(JSON.stringify(source_paths));
+                $("#target_path").val(target_path);
+                $('#move-override-confirm-dialog').modal('show');
```

Comment:
I think when the modal is displayed on error, both the Cancel and Proceed buttons are disabled. The user can't take any actions.

Reply:
I still see this issue. You can reproduce using the following steps:
1. Create a new resource. Create a folder. Upload a file to that folder.
2. Upload the same file to the root of the resource. Then move the file to the folder you created in step 1. When prompted to overwrite, select "Proceed". This works as expected.
3. Now upload the same file to root folder again. Try to move this uploaded file to the folder. You will see the prompt dialog with both buttons disabled.